src/nomad_simulations/schema_packages/properties/__init__.py

```diff
@@ -1,14 +1,12 @@
 from .band_gap import ElectronicBandGap
 from .band_structure import ElectronicBandStructure, ElectronicEigenvalues, Occupancy
 from .energies import (
-    EnergyContribution,
-    FermiLevel,
     KineticEnergy,
     PotentialEnergy,
     TotalEnergy,
 )
 from .fermi_surface import FermiSurface
-from .forces import BaseForce, ForceContribution, TotalForce
+from .forces import TotalForce
```

Comment:
Have you tested the Force or Energy contributions with the new implementation?

Reply:
Yes, though the tests have also been updated.